devtools_m1/cypress/integration/CustomerCartInteractions.feature

```diff
@@ -58,3 +58,27 @@ Feature: Customer Cart Interactions
     Then A bundle cart event should be sent to Drip
     When I check out
     Then A bundle order event should be sent to Drip
+
+  Scenario: A customer adds a simple product to their cart when only that store is configured for Drip
+    Given I am logged into the admin interface
+      And I have set up a multi-store configuration
+      And I have configured Drip to be enabled for 'site1'
+      And I have configured a simple widget for 'site1'
+    When I open the 'site1' homepage
+      And I create an account
+      And I add a 'simple' widget to my cart
+    Then A simple cart event should be sent to Drip
+    When I check out
+    Then A simple order event should be sent to Drip
+
+  Scenario: A customer adds a simple product to their cart when a different store is configured for Drip
+    Given I am logged into the admin interface
+      And I have set up a multi-store configuration
+      And I have configured Drip to be enabled for 'site1'
+      And I have configured a simple widget for 'main'
```

Comment:
Maybe this is obvious with more context but `main` doesn't really tell me what this `simple widget` is configured for? Is is a store or a site? Or something else?

Reply:
Makes sense. I'll think about this one… In these tests, it's a store.

Reply:
I'm going to leave this as is for now and keep it in mind for a future refactor. I fear I'd end up in a rabbit hole trying to figure out decent naming, and I want to continue making progress. It's an easy change later.